glass/shells.py

```diff
@@ -538,6 +631,21 @@ def partition_nnls(
     Uses the ``nnls()`` algorithm from ``scipy.optimize`` and thus
     requires SciPy.
 
```

Comment:
This isn't actually true, which took a lot of effort :)

Reply:
Nice, have removed that sentence